### PR TITLE
Routing features

### PR DESCRIPTION
**Pull request by MrNiceRicee**
Routing, still needs to be finished and fleshed out
Goals
- Singular place for routing
- Change the configrouting to match your localhost
- It will have recipes and reviews

### CODE REVIEW

File: src/component/user/UserProfileStart.jsx [hunk 1]
```diff
@@ -51,7 +66,7 @@ const UserProfileStart = () => {
               Dietary Preference
             </Typography>
             <Typography variant="body1">
-              {logStatus.dietaryPrefs || 'No preferences, a food enjoyer'}
+              {displayPrefs}
```
Comment (aleceche) [inline src/component/user/UserProfileStart.jsx]: did we lose the default value here?

File: src/main.jsx [hunk 2]
```diff
@@ -6,6 +6,9 @@ import 'firebase/auth';
 
 import App from './component/App';
 import config from './config/config';
+import test from './routing/routingtest';
+
+test();
```
Comment (aleceche) [inline src/main.jsx:11]: make note to comment this out/remove in production

**Review by aleceche — APPROVED** [review]
We love routing, will have to go over these with everyone during standup

File: src/component/user/UserProfileStart.jsx (hunk 1 above)
Comment (MrNiceRicee) [reply src/component/user/UserProfileStart.jsx]: Oh yeah, let me add the default value in

File: src/main.jsx (hunk 2 above)
Comment (MrNiceRicee) [reply src/main.jsx:11]: noted it on trello

**Comment by MrNiceRicee** [comment]
Added a error check to make sure we don't crash if there are no dietary prefs

**Pull request merged**